### ISSUE TITLE
Select menu document body example doesn't work with a screen reader

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#6150]
**Describe the issue. What is the expected and unexpected behavior?**
When testing the Select menu document body example, I noticed that I couldn't get into the menu with VO, JAWS, or NVDA. I suspect it's because the menu appears in a different location in the DOM as the input. We should investigate how to make this example accessible as there are likely users following our example.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Go to our [select menu document body example](https://www.patternfly.org/v4/components/select/#select-menu-document-body) and try to test with any screen reader.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
bug

**Comment by nicolethoen** [comment]
@mcarrano can we add this to a future milestone?

**Comment by nicolethoen** [comment]
@thatblindgeye not sure if you knew this comment existed on a different, similar issue: https://github.com/patternfly/patternfly-react/issues/6824#issuecomment-1018568358
